config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -106,33 +106,6 @@ periodics:
     testgrid-dashboards: sig-node-containerd
     testgrid-tab-name: containerd-build-test-images
     description: "builds test images for development in progress branch of upstream containerd"
-- interval: 1h
-  name: ci-containerd-e2e-cos-gce
```

Comment:
This job is same is `ci-cri-containerd-e2e-cos-gce`
https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-node/containerd.yaml#L376